README.md

```diff
@@ -1,56 +1,47 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio Setup finish mobile version 
 
-> Description the project.
+> Microverse project and it involved the following.
+- Creating a repository.
+- setting linters.
+- Following Github workflows.
+- Continauation of the portfolio setup mobile version project
 
 
 ## Built With
 
 - Major languages
-- Frameworks
+ - HTML
+ - CSS
 - Technologies used
+ - Github
+ - VS Code
+ - Gitbash
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
-### Setup
+- Internet
+- Browsers
```

Comment:
-Kindly make your READme more descriptive by adding the necessary information that will help anyone that might want to use your code. 👇
![readme](https://user-images.githubusercontent.com/73854999/198695811-0b4844cb-0c0e-419c-9ae3-6b9bae4f34fa.PNG)